### PR TITLE
cgame: load IQM human model

### PR DESCRIPTION
**Pull request by illwieckz**
Load IQM human model.

Requires:

- https://github.com/UnvanquishedAssets/res-players_src.dpkdir/pull/19

### CODE REVIEW

File: src/cgame/cg_animdelta.cpp [hunk 1]
```diff
@@ -71,17 +71,7 @@ bool AnimDelta::ParseConfiguration(clientInfo_t* ci, const char* token2, const c
...
-	if ( !Q_stricmp( ci->modelName, "human_naked"   ) ||
-		!Q_stricmp( ci->modelName, "human_light"   ) ||
-		!Q_stricmp( ci->modelName, "human_medium" ) )
-	{
-		Q_strncpyz( newModelName, "human_nobsuit_common", sizeof( newModelName ) );
-	}
-	else
-	{
-		Q_strncpyz( newModelName, ci->modelName, sizeof( newModelName ) );
-	}
```
Comment (DolceTriade) [inline src/cgame/cg_animdelta.cpp:84]: why is this required?

**Comment by slipher** [comment]
This should go into a for-x.y.z branch so that developers can keep working on master without having to acquire new assets.

Is there supposed to be any user-visible change from this?

**Comment by illwieckz** [comment]
> This should go into a for-x.y.z branch

Yes, I'll change the target later. I wanted to push the branch early so @DolceTriade can put his hands on it and have everything wired up to test any new animation he may implement for the psaw.

> Is there supposed to be any user-visible change from this?

No. Any visible change should be considered a bug before being investigated.

The purpose of those branch is only to convert the file formats and file layout to the asset pipeline we currently have the best support for.

One can also see the benefit of using recommended conventions so we can rely on default loaders, meaning we can drop the dedicated loading implementation.

Migrating the file for the sake of convention is something I wanted to do for years but the purity win was not worth the effort. Now that @DolceTriade is working on models, making the files compatible with a better workflow and working production pipeline is worth the effort.

File: src/cgame/cg_animdelta.cpp (hunk 1 above)
Comment (illwieckz) [reply src/cgame/cg_animdelta.cpp:84]: This code was required when our human files were not following conventions. Now that the files are following convention, this custom loader can be dropped to rely on the generic one instead. In fact we should drop the custom loader to not look for the old files.

This is useless custom code being deleted. This was basically a workaround for the original files not following conventions.

Comment (DolceTriade) [reply src/cgame/cg_animdelta.cpp:84]: we can drop the copy too then

Comment (illwieckz) [reply src/cgame/cg_animdelta.cpp:84]: Done.

**Comment by DolceTriade** [comment]
Let's merge this now.

**Comment by illwieckz** [comment]
I just rebased to run the CI on it, this was 325 commits behind…
